spec/requests/organizer_can_create_event_spec.rb

```diff
@@ -2,16 +2,18 @@
   let(:user) { create(:user) }
   let!(:group) { create(:group) } 
 
-  let!(:events) do 
-    3.times { create(:event, group: group) }
-  end
-
   describe 'POST req with valid credentials' do
     let(:user_credentials) { user.create_new_auth_token }
     let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(user_credentials) }
     
     before do
-      post "/groups/#{group.id}/events", params: { event: {title: 'Craft Academy'} }, headers: headers
+      post "/groups/#{group.id}/events", params: { event: 
+                                                  {title: 'Craft Academy',
+                                                  description: 'Graduation Party',
+                                                  location: 'Stockholm',
+                                                  date: '2019-12-12',
+                                                  time: '2000-01-01T12:12:12.000Z'
```

Comment:
?? Why set time in year 2000?

Reply:
We came across this when working with Oliver.  This is the output it gives us when we put in the correct format.  The test will only pass when we put it in THIS format.  Couldn't figure this one out.
<img width="558" alt="screen shot 2019-02-05 at 8 32 27 pm" src="https://user-images.githubusercontent.com/43846881/52299374-83cab580-2985-11e9-803b-4405b2004b93.png">